content/en/serverless/azure_app_services.md

```diff
@@ -24,61 +24,68 @@ further_reading:
 
 ## Overview
 
-Microsoft [Azure App Service][1] is a group of serverless resources that enable you to build and host web apps, mobile back ends, event-driven functions, and RESTful APIs without managing infrastructure. It can host workloads of all sizes and offers auto-scaling and high availability options.
+Microsoft [Azure App Services][1] is a group of serverless resources that enable you to build and host web apps, mobile back ends, event-driven functions, and RESTful APIs without managing infrastructure. It can host workloads of all sizes and offers auto-scaling and high availability options.
 
 Datadog provides monitoring capabilities for all Azure App Service resource types:
 
 - Azure Monitor metrics for [Apps][2] and [Functions][3] using the [Azure Integration][2].
-- Use the [Azure App Service View][4] (beta) to quickly spot issues, map relationships between your Azure App Service resources, and gain insights into cost and performance.
+- Use the [Azure App Service View][4] to quickly spot issues, map relationships between your Azure App Service resources, and gain insights into cost and performance.
 - Custom metrics can be submitted using the API.
 - [Resource logs][5] can be submitted using [Event Hub][6].
 
-The Datadog extension for Azure App Service provides additional monitoring capabilities for [Azure Web Apps][7]. This support includes:
+The Datadog extension for Azure App Service provides additional monitoring capabilities.
 
 - Full distributed APM tracing using automatic instrumentation.
+- Customized APM service and trace views showing relevant Azure App Service metrics and metadata.
 - Support for manual APM instrumentation to customize spans.
 - `Trace_ID` injection into application logs.
-- Support for submitting custom metrics using [DogStatsD][8].
+- Support for submitting custom metrics using [DogStatsD][7].
 
 ## Setup
 
+{{< tabs >}}
+{{% tab ".NET" %}}
+
 ### Requirements
 
-1. If you haven't already, set up the [Microsoft Azure integration][9] first.
+1. If you haven't already, set up the [Microsoft Azure integration][1] first.
 
-2. The extension only supports Azure App Service Web Apps. **Function Apps are not supported**.
+2. The extension supports the following resource types:
+    - Azure App Service Web Apps
+    - Function Apps hosted on Basic, Standard, Premium, or Isolated App Service plans. 
 
-<div class="alert alert-warning">To be notified when a private beta is available for Function Apps or additional runtimes, <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">sign up here</a>.</div>
+    <div class="alert alert-warning">Support for .NET Function Apps is in beta for extension v2.2+. Function Apps on consumption plans are not supported. There are no billing implications for tracing functions during this period.<br/><br/>Interested in support for other App Service resource types or runtimes? <a href="https://forms.gle/n4nQcxEyLqDBMCDA7">Sign up</a> to be notified when a beta becomes available.</div>
```

Comment:
Function Apps on consumption plans & out-of-process functions are not supported.